.github/workflows/build-deployment-container.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Build deployment container
+on:
+  push:
+    branches:
+      - prod
+      - staging
```

Comment:
not sure `staging` is any use - but meh - also we don't have a `prod` branch but guess that can all come later